backend/kafka-queue/kafkaqueue.go

```diff
@@ -208,6 +208,10 @@ func New(ctx context.Context, topic string, mode Mode, configOverride *ConfigOve
 			CommitInterval:        time.Second,
 			MaxAttempts:           10,
 			WatchPartitionChanges: true,
+			GroupBalancers: []kafka.GroupBalancer{
+				kafka.RackAffinityGroupBalancer{Rack: findRack()},
```

Comment:
if we're not doing it already somewhere else, it would be nice to add the `ReaderConfig` object as part of the "initializing kafka consumer" message so we can tell that the `findRack` is working as expected in prod